waltz-tools/src/test/java/com/wepay/waltz/tools/performance/PerformanceCliTest.java

```diff
@@ -127,4 +127,43 @@ public void testRunConsumer() throws Exception {
         }
     }
 
+    @Test
+    public void testRunConsumerUpToDateWaterMark() throws Exception {
```

Comment:
rename to `testRunConsumerMountFromLatestHighWaterMark`

Reply:
May be better way to test with `--mount_from_latest ` option would be if we test this as part of `testRunConsumer` itself.

So, we can first call RunConsumers without `mount_from_latest` option and this would append some transactions to Waltz cluster. And after that, we can again call RunConsumers with `mount_from_latest` option.  And if this passes successfully then we can assume that the clients have been mounted from the latest.

You can make sure to reset the outContent so that when you are looking for "Read 10 transactions" statement that would be corresponding to the `mount_from_latest` option execution.